src/protocolsupport/zplatform/pe/MCPEServer.java

```diff
@@ -61,6 +61,7 @@ public void executeHandler(Runnable runnable) {
 			}, new UserChannelInitializer() {
 				@Override
 				public void init(Channel channel) {
+					System.out.println("INIT!!!!");
```

Comment:
I forgot to take that out. I wanted to see if there was even a connection being initialised. Not to worry